src/benchmarks/micro/libraries/System.IO.FileSystem/Perf.FileStream.cs

```diff
@@ -66,6 +66,7 @@ public int Read()
         }
         
         [Benchmark]
+        [BenchmarkCategory(Categories.NoWASM)]
```

Comment:
are file operations not supported in general on WASM or just the async ones?

Reply:
File operations do not seem to be allowed on WASM. From what I was able to glean from reading you need to hit a web endpoint if you want to access files.

Reply:
So maybe that would be something we could do long term, but for now we can just turn them off.